kimchi/src/verifier.rs

```diff
@@ -511,6 +547,8 @@ where
         ));
     }
 
+    check_proof_evals_len(proof);
```

Comment:
this needs a `//~ 1.` comment describing what's happening so that the check can be added to the specification here https://o1-labs.github.io/proof-systems/specs/kimchi.html

Reply:
Pls check https://github.com/o1-labs/proof-systems/pull/989/commits/d05bdd1287a6d6d2f767eacbfde2163aa552a250